contents/updates/content-updates.html

```diff
@@ -0,0 +1,45 @@
+---
+title: Updates
+layout: layout-post
+permalink: /updates/
+---
+
+<h2>Migrating to v2</h2>
+<h6>Last updated on July 14th, 2022</h6>
+<p>
+  Yes, we are finally migrating to a new portal soon! The v2 will be released by
+  the end of this quarter. Tune in here for further updates!
+</p>
+<hr />
+
+<h3>v2 Release Info:</h3>
+
+<div class="sgds-box">
+  <ul>
+    <li>
+      <h5>v2 Portal with Bootstrap 5</h5>
+      <p>
+        We will be incorporating Bootstrap 5 as our base framework for our v2
+        components. This will also be leveraged across upcoming projects.
+      </p>
+    </li>
+    <li>
+      <h5>React Storybook Library</h5>
+      <p>
+        In addition to our v2 portal, we will also be releasing a React library
+        version displayed in Storybook UI explorer to cater to more users.
+        Upcoming projects such as Vue & Web Component will also be hosted in
+        Storybook.
```

Comment:
In addition to our v2 portal, we will also be releasing a React library
version displayed in Storybook. Upcoming support for other JS frameworks will be release in the upcoming months